stable/datadog/README.md

```diff
@@ -60,6 +60,7 @@ The following table lists the configurable parameters of the Datadog chart and t
 | `datadog.tags`              | Set host tags                      | `nil`                                     |
 | `datadog.volumes`           | Additional volumes for the daemonset or deployment | `nil`                     |
 | `datadog.volumeMounts`      | Additional volumeMounts for the daemonset or deployment | `nil`                |
+| `datadog.useDockerSocketVolume` | Enable mounting the docker socket in Agent containers | `True`                |
```

Comment:
could we rename it to `useCriSocketVolume`? Starting with 6.6.0 we'll support any CRI-compliant runtime :)